docs/cell_census_storage_and_release_policy.md

```diff
@@ -1,52 +1,73 @@
-# Cell Census storage & release policy
+# Cell Census data storage & release policy
 
-**Last edited**: Dec, 2022.
+**Last edited**: March, 2023.
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED" "MAY", and "OPTIONAL" in this document are to be interpreted as described in [BCP 14](https://tools.ietf.org/html/bcp14), [RFC2119](https://www.rfc-editor.org/rfc/rfc2119.txt), and [RFC8174](https://www.rfc-editor.org/rfc/rfc8174.txt) when, and only when, they appear in all capitals, as shown here.
 
----
+## Definitions
 
-The cell census MUST be stored in the following S3 root bucket:
+* **Cell Census build**: a SOMA collection with the Cell Census data [as specified in the Cell Census schema](https://github.com/chanzuckerberg/cell-census/blob/main/docs/cell_census_schema.md#data-encoding-and-organization). 
+* **Cell Census source h5ads**: the set of h5ad files used for create a Cell Census build.
+* **Cell Census release**: a Cell Census build that is officially and publicly hosted online.
+* **Cell Census release `tag`**:  a label for a Cell Census release, it MUST be a string of printable ASCII characters.
 
-`s3://cellxgene-data-public/`
+## Cell Census release policy
 
-All contents of a Cell Census build data MUST be deposited in a folder named `cell-census`: 
+### Long-term supported Cell Census release
 
-`./cell-census/`
+* It MUST be created and made available for public access every six months. 
+* Its date of publication MUST be used as the `release_date` (see [below](#Cell-Census-release-information)). 
+* All data from CELLxGENE Discover hosted at the `release_date` that meets the requirements [according to the schema](https://github.com/chanzuckerberg/cell-census/blob/main/docs/cell_census_schema.md#data-included) MUST be included in it.
+* It MUST be available for public access for at least 5 years upon `release_date`.
 
-This folder MUST contain all public builds and releases defined as:
+### Weekly Cell Census release (latest)
 
-* Census "build" is a SOMA collection / container containing cell data.
-* Census "RELEASE" is a "build" which has been officially blessed and released in a formal manner.
-* A "build" is named with a `tag`, which is a string of printable ASCII (eg, "foo", "2022-11-01", "1-rc2.9"):
-   `./cell-census/[tag]/`
-* a RELEASE is named with a tag following the conventional format `release-[counter]`:
-   `./cell-census/release-[counter]/`
+* It MUST be created and made available for public access every week on Mondays except for CZI holidays.
```

Comment:
I would make this `SHOULD`, if that's reasonable. I figure either we're fully automating it, in which case holidays don't matter (barring build failures), or we should be less strict about exactly when it appears: e.g. "once per week, best effort"